src/main/java/org/jitsi/jigasi/transcription/AbstractTranscriptPublisher.java

```diff
@@ -162,6 +172,40 @@ protected void saveTranscriptToFile(String fileName, T transcript)
         }
     }
 
+    /**
+     * Create a directory at a specific path if it's not created
+     *
+     * @param path the path as a string of the directory which should be
+     * created
+     * @return True when the directory was created or already exists, false
+     * otherwise
+     */
+    protected static boolean createDirectoryIfNotExist(Path path)
+    {
+        File dir = path.toFile();
+
+        // Try to make the directory
+        if(!dir.exists())
+        {
+            if(!dir.mkdir())
```

Comment:
Perhaps use `mkdirs`?

Reply:
fixed, the folder might indeed not exist yet.